package.json

```diff
@@ -49,6 +49,8 @@
         "add": "lerna add $1 --scope=@padloc/$scope",
         "remove": "rm packages/$scope/package-lock.json && lerna exec \"npm uninstall $1\" --scope=@padloc/$scope",
         "prettier": "prettier --write .",
-        "prettier:check": "prettier --check ."
+        "prettier:check": "prettier --check .",
+        "update-version": "lerna version $1",
```

Comment:
Why not add `&& node update-release-version.js $1 $2` here? Then we wouldn't have to call it explicitly every time (I assume the two command would always be run together).

Reply:
Good point, I'll change that.